bindings/hyperdrivepy/src/hyperdrive_state_methods.rs

```diff
@@ -116,6 +116,23 @@ impl HyperdriveState {
         Ok(result)
     }
 
+    pub fn calculate_pool_deltas_after_add_liquidity(
+        &self,
+        contribution: &str,
+    ) -> PyResult<(String, String, String)> {
+        let contribution_fp = FixedPoint::from(U256::from_dec_str(contribution).map_err(|_| {
+            PyErr::new::<PyValueError, _>("Failed to convert contribution string to U256")
+        })?);
+        let (result_fp1, result_fp2, result_fp3) = self
+            .state
+            .calculate_pool_deltas_after_add_liquidity(contribution_fp, true)
+            .unwrap();
```

Comment:
instead of unwrapping, I would call `.map_error()` and map it to a PyError.  I think there is prior art for this in the file.

Reply:
Yeah do something like this:
```
            .map_err(|err| {
                PyErr::new::<PyValueError, _>(format!(
                    "Calculate_targeted_long_with_budget returned the error: {:?}",
                    err
                ))
            })?;
 ```
 
Fix the comment obv, but this will catch the error and forward it instead of panic-ing if the function fails for a known reason.

Reply:
roger